### DISCUSSION TITLE
some audio servers cause editor skipping

### DISCUSSION THREAD

**Discussion by Axilotl17** [ppy/osu#22425] [Q&A] [answered]
### Type

Game behaviour

### Bug description

When using some audio servers on arch linux, the editor skips in the timeline.

### Screenshots or videos

It occurs with pulseaudio and pipewire sound servers

https://user-images.githubusercontent.com/78441708/214647096-0f993882-f4b7-4619-aad2-432b8734d4a3.mp4

however jack connection kit works fine

https://user-images.githubusercontent.com/78441708/214648853-cfc5c01b-d7f6-47e6-8229-be67e656560c.mp4

### Version

dev build, built 1 hour ago (latest commits) 

### Logs

[database.log](https://github.com/ppy/osu/files/10502307/database.log)
[input.log](https://github.com/ppy/osu/files/10502303/input.log)
[performance.log](https://github.com/ppy/osu/files/10502304/performance.log)
[runtime.log](https://github.com/ppy/osu/files/10502305/runtime.log)

**Comment by bdach** [comment]
Downgraded to discussion because linux audio. Anything and everything may be broken in your particular config and I would be none the wiser.

**Comment by Axilotl17** [comment, marked as answer]
Now it has broken with all audio servers, even on a fresh install.

**Reply by Axilotl17** [reply]
seems to be a bluetooth issue

**Comment by Apep-the-snek** [comment]
How did you manage to get Jack working with osu on linux? is there any documentation on this? i can't find it.
